tensorflow/lite/kernels/embedding_lookup_test.cc

```diff
@@ -109,6 +107,39 @@ TEST(EmbeddingLookupOpTest, SimpleTest) {
               })));
 }
 
+TEST(EmbeddingLookupOpTest, VariableShapeTest) {
+  EmbeddingLookupOpModel m({2}, {3, 2, 4});
```

Comment:
I don't think this tests the "variable input shape" in the same way that ahentz@ intended in the TODO. Google "TensorFlow dynamic shape" and that should be what he was referring to. I'm not sure if the embedding lookup supports it yet.

This test case itself doesn't add any more coverage than the SimpleTest one, so I'd remove it.

Reply:
@alanchiao , thanks for the input, i have removed the TC.

Regards
Amit